### DISCUSSION TITLE
Kibana: need to create index patterns before use

### DISCUSSION THREAD

**Discussion by arska** [appuio/appuio-cloud-community#33] [Help Needed] [answered]
As a developer user looking at pod logs
When I click on the link "Show in Kibana"
Then I need to "Create index pattern" first, but there is no default or explanation how to

In OCP3 this was AFAIK not necessary?

It seems to be an OCP4 "feature": https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-visualizer.html: "Each user must manually create index patterns when logging into Kibana the first time to see logs for their projects. Users must create an index pattern named app and use the @timestamp time field to view their container logs."

![screencapture-logging-apps-cloudscale-lpg-2-appuio-cloud-app-kibana-2021-12-30-21_04_52](https://user-images.githubusercontent.com/43398/147785030-3896d26b-bb02-417d-ba97-53962afead8a.png)

**Comment by tobru** [comment, marked as answer]
> In OCP3 this was AFAIK not necessary?

It was the same in OCP3.

See https://docs.appuio.cloud/user/how-to/access-logs-through-kibana.html
